src/base/static/components/molecules/form-stage-control-bar.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 import { translate, Trans } from "react-i18next";
 
-import { Link } from "../atoms/typography";
+import { Button } from "../atoms/buttons";
```

Comment:
This is a button, not an `a` tag because it has no href: https://css-tricks.com/use-button-element/#article-header-id-2

It's a subtle difference, but it seems like we should only be using `a` tags when we are navigating to a new route.